### PR TITLE
Is true or false

### PR DESCRIPTION
**Pull request by ufobat**
Added Features to test in boolean context.
This addresses #8 and #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/zoffixznet/perl6-Testo.svg)](https://travis-ci.org/zoffixznet/perl6-Testo)
+l[![Build Status](https://travis-ci.org/zoffixznet/perl6-Testo.svg)](https://travis-ci.org/zoffixznet/perl6-Testo)
```
Comment (AlexDaniel) [inline README.md]: oops?

**Comment by AlexDaniel** [comment]
So you added `is-true`, `is-false`, `is-so`, `is-not`. What's the justification for introducing these? Looks like bloat to me.

**Comment by AlexDaniel** [comment]
Ah… I see, there's no other easy/obvious way to check for Truthy/Falsey values?

**Comment by ufobat** [comment]
If we considder #18 as a duplicate of #8 I would suggest to use `is-so` and `is-not` because `is-true` sounds as if we would do a `$got === True`. `so` and `not` are used to enforce a boolean context.

**Comment by AlexDaniel** [comment]
And there's also https://github.com/perl6-community-modules/perl6-Testo/issues/6 which suggests to rename `is-run` to `runs`… Following that logic, maybe `ok` is not too bad. Don't know if this bikeshedding is worth it, but I do think that more obvious routine names is a good thing in a long run.

**Comment by AlexDaniel** [comment]
I think `ok` and `nok` make sense for now. It's possible that we'll come up with better names for them, but if we want to create a mostly drop-in replacement for Test, I think it's not wrong to have `ok` and `nok`.

Feel free to merge this yourself.

**Pull request merged**